### PR TITLE
new validation

### PR DESCRIPTION
**Pull request by Alvaro-Rodrigueez**

### CODE REVIEW

File: validation/Steady state seepage validation/model.json [hunk 1]
```diff
@@ -0,0 +1,520 @@
+{
+    "id": "36b0804e-3ae5-4f00-95bc-f4df405a77ab",
+    "metadata": {
+        "title": "Test 10a - Dam with core - HARR",
```
Comment (timrutter) [inline validation/Steady state seepage validation/model.json:4]: i think the title and description should be reviewed.  don't think anyone will know what test 10a means.

File: validation/Steady state seepage validation/model.json [hunk 2]
```diff
@@ -0,0 +1,520 @@
...
+        "description": "Conductivity variation tests - Based on Example 8-6",
```
Comment (timrutter) [inline validation/Steady state seepage validation/model.json:5]: and what is example 8-6

**Review by timrutter — APPROVED** [review]
title and description in the model need review but otherwise good.  Can be merged for now if needs be.

File: validation/Steady state seepage validation/model.json (hunk 2 above)
Comment (Alvaro-Rodrigueez) [reply validation/Steady state seepage validation/model.json]: Updated new description

**Pull request merged**